src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { User } from '../User';
+import PropTypes from 'prop-types';
+import { TypeUser } from '../Types/TypeUser';
+import './Todo.scss';
+
+export const Todo = ({ title, completed, user }) => (
+    <div className="todo-item">
+      <div className="todo-title">
+        {title}
+      </div>
+      <div className="todo-status">
+        Status: {completed ? 'completed' : 'not completed'}
+      </div>
+      <User user={user}/>
```

Comment:
You can use spread operator here. It will provide you easier destructuring in `User` component.
```suggestion
      <User {...user} />
```